html-css/week1/mateus_britto/portfolio_page/styles.css

```diff
@@ -0,0 +1,477 @@
+@import url("https://fonts.googleapis.com/css2?family=Anton&family=Poppins:ital,wght@0,300;1,300&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
+html {
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+  font-size: 100%;
+}
+
+body {
+  height: 100vh;
+  position: relative;
+  font-family: "Poppins", sans-serif;
+  background-color: #121212
+}
+
+.nav-bar {
+  width: 100%;
+  height: 4rem;
+  position: fixed;
+  display: flex;
+  align-items: center;
+  justify-content: end;
+  background: #0f0f0f;
+  padding-right: 50px;
+  z-index: 1;
+}
+
+.nav-ul,
+.nav-li {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+  height: 4rem;
+}
+
+.section-link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: antiquewhite;
+  height: 100%;
+  font-size: 1.8rem;
+  padding: 0 20px;
+  transition: background-color 1s ease;
+}
+
+.section-link:hover {
+  background-color: rgba(24, 89, 63, 0.5);
+}
+
+.container {
+  padding-top: 4rem;
+}
+
+#about {
+  height: 100vh;
```

Comment:
If you resize your browser in a way that the height is very narrow, then some of the content would be cropped. It's better if you change this to:
```suggestion
  min-height: 100vh;
```
instead.
![cropped-text](https://github.com/user-attachments/assets/27f1d137-b423-4b1c-bddb-8e7fbcb1cfa3)